Project.toml

```diff
@@ -10,11 +10,13 @@ Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
```

Comment:
If only `splitdef` is needed from MacroTools, I suggest using [ExprTools.jl](https://github.com/invenia/ExprTools.jl) instead. It's a more light-weight package without additional dependencies that contains `splitdef` and `combinedef`.

Reply:
I also use `@capture`.